src/contexts/Localisation.tsx

```diff
@@ -118,21 +125,53 @@ export function LanguageProvider({
     }
   }
 
+  // If the selected language is no longer valid (either disabled or uninstalled), then try default, or fallback to first available
+  const getValidLanguageCode = () => {
+    if (languageOptions.some((lang) => lang.enabled && lang.code === selectedLanguageCode))
+      return selectedLanguageCode
+    if (languageOptions.some((lang) => lang.enabled && lang.code === defaultLanguageCode))
+      return defaultLanguageCode
+    else {
+      console.log('Invalid language code, falling back to first available')
```

Comment:
Maybe good to add which is the first available code?

Reply:
Sure: 96a1c966